public/static/docs/command-reference/gc.md

```diff
@@ -33,15 +33,15 @@ restored using `dvc fetch`, as long as they have previously been uploaded with
 
 ## Options
 
-- `-a`, `--all-branches` - keep cached objects referenced from the latest commit
-  across all Git branches. It should be used if you want to keep data for the
-  latest experiment revisions. Especially, if you intend to use `dvc gc -c` this
-  option is much safer.
+- `-a`, `--all-branches` - keep cached objects referenced from the latest
+  [revisions](https://git-scm.com/docs/revisions) across all Git branches (tip
```

Comment:
can we just keep it simpler? or at least simple, human readable version first ... formal explanation (only if it's needed - later)

Reply:
OK, simplified it now.